library/rendering/entity_rendering/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	name = "Quilt Entity Rendering API"
+	moduleName = "entity_rendering"
+	id = "quilt_entity_rendering"
+	description = "Quilt APIs relating to the rendering of entities."
+	library = "entity_rendering"
+	moduleDependencies {
+		core {
+			api("qsl_base")
+			testmodOnly("resource_loader")
```

Comment:
Is resource loader as testmodOnly good? (this is a question to anyone)

Since you're most likely to use this for custom resources, having resource loader would be necessary.
Making it an api dependency would make this... I'm not sure how to word it but like, foolproof?

Reply:
> Is resource loader as testmodOnly good? (this is a question to anyone)
> 
> Since you're most likely to use this for custom resources, having resource loader would be necessary. Making it an api dependency would make this... I'm not sure how to word it but like, foolproof?

Yes IMO, resource loader should be a hard api dependency for this api